yt_dlp/extractor/playsuisse.py

```diff
@@ -215,26 +225,15 @@ def _perform_login(self, username, password):
         if not self._ID_TOKEN:
             raise ExtractorError('Login failed')
 
-    def _get_media_data(self, media_id):
-        # NOTE In the web app, the "locale" header is used to switch between languages,
-        # However this doesn't seem to take effect when passing the header here.
-        response = self._download_json(
-            'https://www.playsuisse.ch/api/graphql',
-            media_id, data=json.dumps({
-                'operationName': 'AssetWatch',
-                'query': self._GRAPHQL_QUERY,
-                'variables': {'assetId': media_id},
-            }).encode(),
-            headers={'Content-Type': 'application/json', 'locale': 'de'})
-
-        return response['data']['assetV2']
-
     def _real_extract(self, url):
         if not self._ID_TOKEN:
             self.raise_login_required(method='password')
 
         media_id = self._match_id(url)
-        media_data = self._get_media_data(media_id)
+        query = parse_qs(url)
+        locale_param = (query.get('locale') or ['de'])[0].lower()
+        locale = locale_param if locale_param in {'de', 'fr', 'it', 'rm'} else 'de'
+        media_data = self._get_media_data(media_id, locale)
```

Comment:
```suggestion
        media_data = self._get_media_data(media_id, traverse_obj(parse_qs(url), ('locale', '0'), default='de'))
```

Reply:
why was this marked as resolved?

Reply:
I've implemented your code suggestion and marked it as resolved.
After using this modified version, I noticed that the extraction of metadata for `locale` value did not work anymore: I always got the metadata “de” (german), regardless of of the `locale` value. That's why I reverted that change. See https://github.com/yt-dlp/yt-dlp/pull/12466/commits/291ea778c393c0f3c646c32f10d16ae0ced63a6a

Reply:
oh, I made a typo in my suggestion. It needs to be this:

```suggestion
        media_data = self._get_media_data(media_id, traverse_obj(parse_qs(url), ('locale', 0), default='de'))
```

Reply:
in the future, if there is an issue with a code suggestion, please reply to the review comment instead of reverting and resolving